src/components/Autocomplete/Autocomlete.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { SyntheticEvent, useState } from 'react';
+import Autocomplete from '@mui/material/Autocomplete';
+import TextField from '@mui/material/TextField';
+import debounce from 'lodash.debounce';
```

Comment:
Install `lodash`. Use individual imports - like `import debounce from 'lodash/debounce';`
Also, reinstall types for all library